Assets/Plugins/Source/EOSManager.cs

```diff
@@ -419,6 +419,11 @@ private void InitializeOverlay(IEOSCoroutineOwner coroutineOwner)
             // those cases, this code will early out.
             public void Init(IEOSCoroutineOwner coroutineOwner)
             {
+#if !UNITY_EDITOR && !(UNITY_STANDALONE_WIN)
```

Comment:
nit: Is the extra parenthesis needed here?

Reply:
That's there because the parenthesized section will expand with more supported platforms.